### DISCUSSION TITLE
how is limb darkening handled in phoebe?

### DISCUSSION THREAD

**Discussion by dingxu6207** [phoebe-project/phoebe2#800] [Questions]
I don’t know which parameter represents limb darkening in Phoebe.  how is limb darkening handled in phoebe?

print(b.filter(context='component', kind='star', component='primary'))
ParameterSet: 17 parameters
         requiv@primary@component: 1.5 solRad
C    requiv_max@primary@component: 1.6724563972838384 solRad
C    requiv_min@primary@component: 1.2725418568681297 solRad
           teff@primary@component: 6000.0 K
C          logg@primary@component: 4.089736163094955
        syncpar@primary@component: 1.0
C        period@primary@component: 0.5 d
C          freq@primary@component: 12.56637 rad / d
      gravb_bol@primary@component: 0.32
   irrad_frac_refl_bol@primary...: 0.6
C  irrad_frac_lost_bol@primary...: 0.4
    ld_mode_bol@primary@component: lookup
    ld_func_bol@primary@component: logarithmic
   ld_coeffs_source_bol@primar...: auto
C          mass@primary@component: 1.0089067994531355 solMass
C           sma@primary@component: 1.675 solRad
C         asini@primary@component: 1.675 solRad

**Comment by Raindogjones** [comment]
Hi Ding,
This page might help explain - http://phoebe-project.org/docs/latest/tutorials/limb_darkening

**Reply by dingxu6207** [reply]
Many thanks!
